autopush/logging.py

```diff
@@ -60,15 +60,18 @@
 began_logging = False
 
 
-def begin_or_register(observer):
-    # type: (Any) -> None
+def begin_or_register(observer, redirectStandardIO=False, **kwargs):
+    # type: (Any, bool, **Any) -> None
     """Register observer with the global LogPublisher
 
     Registers via the global LogBeginner the first time called.
     """
     global began_logging
     if not began_logging:
-        globalLogBeginner.beginLoggingTo([observer], redirectStandardIO=False)
+        globalLogBeginner.beginLoggingTo(
+            [observer],
+            redirectStandardIO=redirectStandardIO,
+            **kwargs)
```

Comment:
Do we want to drop the ) when we immediately drop-line after starting, or keep it this way? I'm fine with either way, just want to know which one we should use everywhere.